docs/governance/community-guidelines/ways-of-working.md

```diff
@@ -0,0 +1,32 @@
+# Ways of Working
+
+This document is a [community guideline].
+
+The document outlines the general workflow and describes expected interactions between various working groups ([WG]s).
+
+**Community-Driven Direction:**
+   - The project's direction will be influenced by community needs.
+   - Requests will be processed by the [Communications WG].
+   - The [Communications WG] will collaborate with the [SC] to establish the project's high-level roadmap.
```

Comment:
Whats the difference between this HL Roadmap and the content discussed below?

Reply:
The HL roadmap is the same as the roadmap, dropped the HL part from here. Again, not sure if that answers your question,

Reply:
Thanks. I believe the roadmap would fall under "technical details" insofar as it is a plan for how and when the projects will hit specific targets. That'd be out of scope for the SteerCo.

Reply:
Yeah this was my version of the alternative project drivers in the absence of input from the community as you mentioned below.